### ISSUE TITLE
Assets not displayed in Purview

### ISSUE THREAD

**Issue by vbbastian** [microsoft/Purview-ADB-Lineage-Solution-Accelerator#154]
**Describe the bug**
I installed the demo version of ADB Solution accelerator, after running notebook, I can see successful invocation happening for the PurviewOut Function with valid openai lineage json.  But for some reason i am not getting the assets in Purview. I got the assets once  for the very first run after installation. I deleted them to do some testing and tried rerunning notebook multiple times, but the assets are not showing up again

.**Steps Taken**

1. Confirmed that configuration of Az Functions has green ticks next to Key Vault rreference
2. Tried installing custom types on Purview using PS and got message. that the types are already installed 

**Function App Log**

union traces
| union exceptions
| where timestamp > ago(30d)
| where operation_Id == 'cedb60b36ff487b012d019b07fdfc6d0'
| where customDimensions['InvocationId'] == '56dc42ff-be09-4353-a0c3-f4113bc496d8'
| order by timestamp asc
| project
    timestamp,
    message = iff(message != '', message, iff(innermostMessage != '', innermostMessage, customDimensions.['prop__{OriginalFormat}'])),
    logLevel = customDimensions.['LogLevel']

![screenshot](https://user-images.githubusercontent.com/124236605/216248203-6ac8c93c-8b78-4f4e-b868-fc086ef44bdd.jpg)

**Comment by vbbastian** [comment]
If it helps troubleshooting faster, I am within Microsoft Tenant and can share the artifacts with team.

**Comment by wjohnson** [comment]
#TakenOffline

**Comment by danielfilhocoelho** [comment]
Any updates here? I have a similar problem - I can also see the data go the EH, but it doesn't reach purview. It seems that it always falls on the enrichedEvent == null conditional.
<img width="776" alt="image" src="https://user-images.githubusercontent.com/18191252/225308836-9845f043-1e8e-402d-8c40-085132c0a767.png">
<img width="831" alt="image" src="https://user-images.githubusercontent.com/18191252/225309660-1df8630c-5870-4726-8c4d-a25ac3844362.png">

**Comment by victoradmc** [comment]
Same thing happening to me as @danielfilhocoelho  described

**Comment by fhmessas** [comment]
Same thing happening here, as @danielfilhocoelho

**Comment by jg079963** [comment]
Seeing the same issue.

**Comment by alvintai78** [comment]
Yeah i am having the same issue, not sure has this been fix ? All my deployment is successful without any error message. But i cannot see databricks as an asset inside purview.

**Comment by wjohnson** [comment]
Adding additional logs to help with debugging this (as in #225)
